scala-steward.conf

```diff
@@ -1,15 +1,14 @@
+# At most we want pull requests for a dependency/group no more than once a week.
+# This is to balance responsiveness with dependency management fatigue.
+pullRequests.frequency = "7 days"
```

Comment:
I *think* this works with groupings as expected. If not, the alternative solution is to simply schedule the workflow to run only once a week.